src/components/feature/FeatureProfile.js

```diff
@@ -12,6 +12,33 @@ const FeatureProfile = () => {
 
     const { name, data } = featureProfile
 
+    const getContent = () => {
+        if (Object.keys(data).length === 0) {
+            return (
+                <div className={styles.noData}>
+                    {i18n.t('No data to show for this feature.')}
+                </div>
+            )
+        }
+
+        return (
+            <div className={styles.featureData}>
+                <div className={styles.dataTable}>
+                    <table>
+                        <tbody>
+                            {Object.keys(data).map((key) => (
+                                <tr key={key}>
+                                    <th>{key}</th>
+                                    <td>{data[key]}</td>
+                                </tr>
+                            ))}
+                        </tbody>
+                    </table>
+                </div>
+            </div>
+        )
+    }
+
```

Comment:
Extracting to its own `DrawerContent` component would be a bit cleaner and in this case it'd be quite easy to do because the component would only need the `data` prop.
But this is not critical at all.